odinaka-joy/teamwork-api
# fixes issue#3 Setup-test-frameworks and CI/CD

issue#5 Adds mocha test framework, chai and sinon for TDD #6

## Related Issue
Fixes #3 

### What does this PR do?
Setup TDD and CI/CD Development

### Description of Task to be completed?
Set up eslint following Airbnb style guide
Set up mocha, chai and sinon for TDD
Set up travis-ci, code-climate and coveralls for CI/CD

### How should this be manually tested?
Clone the repo, cd into project folder and run npm install
Run npm run lint to check linting
Run npm run test to run unit test
Create a pull request for travis to check code for error and give feedback

### What are the relevant github project card note?
https://github.com/odinaka-joy/teamwork-api/projects/1#card-28677387
